test/Dapr.Client.Test/InvokeBindingApiTest.cs

```diff
@@ -147,6 +147,66 @@ public async Task InvokeBindingAsync_WithRequest_ValidateRequest()
                 });
         }
 
+        [Fact]
+        public async Task InvokeBindingAsync_WithCustomRequest_ValidateRequest()
+        {
+            await using var client = TestClient.CreateForDaprClient();
+
+            var data = new InvokeRequest() { RequestParameter = "Hello " };
+            var metadata = new Dictionary<string, string>
+            {
+                { "key1", "value1" },
+                { "key2", "value2" }
+            };
+
+            var request = await client.CaptureGrpcRequestAsync(async daprClient =>
+            {
+                return await daprClient.InvokeBindingAsync<InvokeRequest, Widget>("test","create", data, metadata);
+            });
+
+            var resp = new Autogen.Grpc.v1.InvokeBindingResponse();
+            resp.Data = ByteString.CopyFrom(JsonSerializer.SerializeToUtf8Bytes(new Widget() { Color = "red", }, client.InnerClient.JsonSerializerOptions));
+            var response = await request.CompleteWithMessageAsync(resp);
+
+            var envelope = await request.GetRequestEnvelopeAsync<InvokeBindingRequest>();
+            envelope.Name.Should().Be("test");
+            envelope.Metadata.Count.Should().Be(2);
```

Comment:
Should this also verify the binding operation? (`create`)

Should this also verify the body of the request? It's checking the response, but not the request.